spec/controllers/tenant_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Account::TenantsController, type: :controller do
+  before :each do
+    @user = create(:user)
+    sign_in @user
+  end
+  let!(:tenant) { FactoryBot.create(:tenant) }
+
+  it 'Get#index' do
+    get :index
+    expect(response).to have_http_status(200)
+  end
+
+  it 'Get#show' do
```

Comment:
change to "GET #show", similar to other specs

Reply:
done